tests/test_save_load.py

```diff
@@ -2,6 +2,7 @@
 Tests for saving and loading a simple class.
 """
 
+import os
 import tempfile
```

Comment:
why not [pathlib](https://docs.python.org/3.7/library/pathlib.html?highlight=path#module-pathlib)?
``` python
from pathlib import Path
path = Path("folder", "file")
```

Reply:
(I know its just tests, but it is imo much nicer)

Reply:
Yeah, I could just change the `sample` fixture to return a `pathlib.Path` (maybe rename it to `sample_dir`), and then do

`sample_dir / 'folder' / 'file'`

Reply:
Done.